Client/Application/NavigationRouter.swift

```diff
@@ -89,7 +90,12 @@ enum NavigationPath {
         } else if urlString.starts(with: "\(scheme)://fxa-signin"), components.valueForQuery("signin") != nil {
             self = .fxa(params: FxALaunchParams(query: url.getQuery()))
         } else if urlString.starts(with: "\(scheme)://open-url") {
-            let url = components.valueForQuery("url")?.asURL
+            var url: URL?
+            if components.query?.contains("clipboard") ?? false {
+                url = UIPasteboard.general.url
+            } else {
+                url = components.valueForQuery("url")?.asURL
+            }
```

Comment:
Debating if we should clean this up @noorhashem what do you think? 
I am good either ways. 
```suggestion
url = (components.query?.contains("clipboard") ?? false) ? UIPasteboard.general.url : components.valueForQuery("url")?.asURL
```

Reply:
@nbhasin2 this is redundant code actually, it's outdated. I'll update it.